src/deepak/CodingExam3.java

```diff
@@ -0,0 +1,29 @@
+package deepak;
+
+public class CodingExam3 {
+	
+	void findDuplicates(int[] arr) {
+		
+		for(int index=0; index<arr.length; index++) {
+			
+			for(int j=index+1; j<arr.length;j++) {
```

Comment:
Do not write logic in main method.
{ 10, 12,32, 55, 32, 17, 12, 32 } try this input. code wont work as expected